resources/views/bread/order.blade.php

```diff
@@ -50,7 +34,7 @@
 <script>
 $(document).ready(function () {
     $('.dd').nestable({
-        maxDepth: 1
+        @if(empty($nestable)) maxDepth: 1 @endif
```

Comment:
Why check `empty()` here when you return a boolean from the controller?

Reply:
I'm trying to avoid problems in case someone already overrode order method and updates.
Following that logic I forgot to do the same in the partial view. 
Let me know if you prefer instead that we force the user to keep controller updated.